README.md

```diff
@@ -11,7 +11,7 @@ Refract is a recursive data structure for expressing complex structures, relatio
 
 ## Version
 
-**Current Version**: 0.4.0
+**Current Version**: 0.4.1
```

Comment:
I think this should be a minor version bump because there are new elements. (0.5.0)